### PR TITLE
Add login page

### LINKED ISSUE #28: Add separate login page
Maybe have a simple web page at ```/login``` that has a simple layout with our logo and a "sign in with Google" button that handles signing in and out of the Copilot client. Having the weird pop up in front of all of the placeholder loading text of the messenger client looks really bad.

### PR DESCRIPTION
**Pull request by gmittal**
Closes #28. Looks pretty good. The web copy is still placeholder and needs to be fixed (cc: @ankitr).

@ankitr PTAL.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -21,6 +24,21 @@ app.get('/js/main.js', (req, res) => {
   });
 });
 
+app.get('/js/login.js', (req, res) => {
```
Comment (ankitr) [inline app.js:27]: Is this idiomatic? I'm not too familiar with how one configures local javascript.

File: static/login.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
+    <link rel="shortcut icon" href="/favicon.png">
+    <script src="//www.gstatic.com/firebasejs/ui/live/0.4/firebase-ui-auth.js"></script>
+    <link type="text/css" rel="stylesheet" href="//www.gstatic.com/firebasejs/ui/live/0.4/firebase-ui-auth.css" />
+    <link href="/css/login.css" rel="stylesheet" type="text/css">
+    <title>Copilot Volunteer Login</title>
+  </head>
+  <body>
+    <a href=//www.copilot.help/><h1 id=logo aria-label=Copilot>Copilot</h1></a>
+    <p>Volunteers are the core of our team. This is your chance to help students in need. <ins>This website lets you anonymously support peers who are going through tough times and make sure they get the help they need. <a href="mailto:team@copilot.help">Contact us</a> for an invitation to our growing team.</p>
```
Comment (ankitr) [inline static/login.html]: Can you style this so that the link looks good?

File: static/login.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    <p>Welcome to the team. <ins>Sign in with Google to continue.</ins></p>
+    <div id="login" class="loginBtn google">Login with Google</div>
```
Comment (ankitr) [inline static/login.html]: Make this button span the size of the text, not the width of the container. I think that would look better.

File: static/login.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```
Comment (ankitr) [inline static/login.html]: Do we need jQuery for this?

**Review by ankitr — CHANGES_REQUESTED** [review]
Great direction. A little feedback and a few questions, but should be good to merge soon.

File: app.js (hunk 1 above)
Comment (gmittal) [reply app.js:27]: Manually replacing config variables as client-side JS is served is a common practice. Though they often use tools like https://github.com/yahoo/express-state to do it. Perhaps I can add this in a future release.

File: static/login.html (hunk 4 above)
Comment (gmittal) [reply static/login.html]: I'm using it for one small thing in ```js/login.js```. I can get rid of it and use vanilla JS too though.

File: static/login.html (hunk 2 above)
Comment (gmittal) [reply static/login.html]: Not sure if I follow. The link is currently just our logo.

File: static/login.html (hunk 3 above)
Comment (gmittal) [reply static/login.html]: The button is same width as the box of text.

Comment (gmittal) [reply static/login.html]: Also, I'm going to try and get a hold of the official login with Google button.

File: static/login.html (hunk 2 above)
Comment (gmittal) [reply static/login.html]: Oh got it.

File: static/login.html (hunk 3 above)
Comment (gmittal) [reply static/login.html]: Fixed.

File: static/login.html (hunk 4 above)
Comment (gmittal) [reply static/login.html]: Fixed.

File: static/login.html (hunk 2 above)
Comment (gmittal) [reply static/login.html]: Fixed.

File: app.js (hunk 1 above)
Comment (ankitr) [reply app.js:27]: Ok, sounds good.

**Review by ankitr — APPROVED** [review]
LGTM

**Pull request merged**